### PR TITLE
Company-db closes #31 and closes #36

### PR DESCRIPTION
**Pull request by bhargavisatpathy**
Changed db from sqlite to pg
Included create action in company_controller,rb
Put in flash notices in application.html.erb
Wrote company controller tests

### CODE REVIEW

File: app/views/layouts/_flash.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<% flash.each do |name, message| %>
+  <div class="<%= name %>">
+    <p><%= (message).html_safe %></p>
```
Comment (oorion) [inline app/views/layouts/_flash.html.erb]: what's the rationale for using html_safe here?

File: config/database.yml [hunk 2]
```diff
@@ -5,21 +5,21 @@
 #   gem 'sqlite3'
 #
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
```
Comment (cluhring) [inline config/database.yml:8]: postgres is my jam

**Comment by oorion** [comment]
Everything looks great!

File: spec/controllers/companies_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+                organization: 'Google',
+                       title: 'RoR',
+                       state: 'CO',
+                        city: 'Denver',
+                       email: 'google@email.com'
+                       }
+    post :create, company: company_params
+    assert_redirected_to root_path
+    expect(assigns(:company)).not_to be_nil
+    expect(assigns(:company)).to be_a(Company)
```
Comment (cluhring) [inline spec/controllers/companies_controller_spec.rb:17]: rspec is weird - not always easy to understand.

File: spec/factories.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+FactoryGirl.define do
```
Comment (cluhring) [inline spec/factories.rb:1]: You still need to show me how to create a random person using FactoryGirl, ok?

File: app/views/layouts/_flash.html.erb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<% flash.each do |name, message| %>
```
Comment (cluhring) [inline app/views/layouts/_flash.html.erb:1]: watch for dublicate - Alex has flash in her PR

File: app/views/layouts/_flash.html.erb (hunk 1 above)
Comment (bhargavisatpathy) [inline app/views/layouts/_flash.html.erb]: Just keeping an extra set of eyes...can take off if u want :)

**Pull request merged**